dist/angular-patternfly.js

```diff
@@ -611,6 +613,8 @@ angular.module('patternfly.card').directive('pfCard', function () {
             'total': '1000'
         };
 
+       $scope.availLabel = "available";
```

Comment:
lol..'availLabel' is actually correct, consistent with 'usedLabel', 'pctLabel', and 'noLabel'.